src/main/kotlin/kweb/Kweb.kt

```diff
@@ -295,147 +315,137 @@ class Kweb private constructor(
                             | requires JavaScript to be enabled.""".trimMargin())
             }
         }
-        for (plugin in plugins) {
-            applyPluginWithDependencies(plugin = plugin, appliedPlugins = mutableAppliedPlugins, document = docTemplate, routeHandler = this)
-        }
 
         return { docTemplate.clone() }
     }
 
-    private fun Routing.listenForWebsocketConnection(path: String = "/ws") {
-        webSocket(path) {
-
-            val hello = gson.fromJson<Client2ServerMessage>(((incoming.receive() as Text).readText()))
-
-            if (hello.hello == null) {
-                error("First message from client isn't 'hello'")
-            }
+    private suspend fun DefaultWebSocketSession.listenForWebsocketConnection() {
```

Comment:
This function has not changed at all except extracting the websocker outside for increased visibility, but because of the formatting it turns up in the diff